taskcluster/scripts/toolchain/cross-compile-setup.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
I don't feel qualified to review this setup. I guess it's the right one.

Reply:
Me neither. As far as I can tell, copy-paste from decisionlib so it should be fine O_O